src/main/java/com/cleanroommc/groovyscript/compat/mods/additionalenchantedminer/AdditionalEnchantedMiner.java

```diff
@@ -0,0 +1,7 @@
+package com.cleanroommc.groovyscript.compat.mods.additionalenchantedminer;
+
+import com.cleanroommc.groovyscript.compat.mods.GroovyPropertyContainer;
+
+public class AdditionalEnchantedMiner extends GroovyPropertyContainer {
+    public final WorkBenchPlus WorkBenchPlus = new WorkBenchPlus();
```

Comment:
fields should be in `camelCase` - `WorkBenchPlus` is PascalCase, it should be `workBenchPlus`